### PR TITLE
Add acceptance to ci

### PR DESCRIPTION
**Pull request by t-keazirian**
Added running acceptance tests to CI

- removed one `@wip` to test it is working
- Running the server in the background with `&`

[Trello Ticket](https://trello.com/c/I3DgXkDd/34-add-acceptance-tests-to-ci-pipeline)

### CODE REVIEW

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -22,3 +22,16 @@ jobs:
...
+          ruby-version: '2.5.1'
+          working-directory: './http_server_spec'
+      - name: Bundle Ruby
+        working-directory: './http_server_spec'
+        run: bundle install
+      - name: Run Server in background
+        run: dotnet run --project TKeazirian.HTTPServer &
+      - name: Run the Ruby Tests
+        working-directory: './http_server_spec'
+        run: rake test
```
Comment (Aoxian) [inline .github/workflows/dotnet.yml:37]: Is there a way to group this a little differently?  Now that you have it working, I do not think we need to see everything so granular.

```text
> Build
> Unit Tests
> Acceptance Tests
> Post Run actions.checkout@v3
> Complete Job
```
image

Comment (t-keazirian) [reply .github/workflows/dotnet.yml]: Check it out now!  Should be more condensed.

Comment (Aoxian) [reply .github/workflows/dotnet.yml]: Looks Great!

**Review by Aoxian — COMMENTED** [review]
Let's clean up the git history, and this one should be good to go.

**Comment by t-keazirian** [comment]
> Let's clean up the git history, and this one should be good to go.

Did I do something wrong or is it expected that the one commit is there now (after my rebase), but if you click on the ellipses, then all the other squashed commits are below it?

**Pull request merged**